src/kibana/components/vislib/lib/legend.js

```diff
@@ -17,16 +20,21 @@ define(function (require) {
      * @param color {Function} Color function
      * @param _attr {Object|*} Reference to Vis options
      */
-    function Legend(vis, el, labels, color, _attr) {
+    function Legend(vis) {
```

Comment:
Don't forget to update the doc block to remove the attributes this is no longer expecting